.github/workflows/continuous-integration.yaml

```diff
@@ -9,6 +9,33 @@ on:
       - 'alpha-release/**'
 
 jobs:
+  autopep8:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
I think maybe somewhere we should document that we are very strict on pep8 and ask people to run it themselves locally before making a PR so we don't get flooded with duplicated PRs.
This could go into some kind of CONTRIBUTING.md (which should also explain not to edit the generated code).

Reply:
I would suggest adding this to the ways of working page of the docs (maybe in a new Python code style section) and then linking that from the README, rather than creating another file. We won't be flooded with duplicate PRs straight away as we'll need to approve the action run for any PRs from external contributors.

Reply:
Yeah I think linking ways of working in the README and adding it there may be the best approach

Reply:
I'm currently working on the documentation for it, and i will be including the why and how to run the checks locally. Once that has been checked over I'll add it to the README & docs